index.html

```diff
@@ -0,0 +1,187 @@
+        <h4>What we do</h4>
+        <div class="program-line"></div>
+      </div>
+
+      <div class="card-container">
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            computer
+          </span>
+            <p class="card-margin technology">Repairs</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            tools_installation_kit
+          </span>
+            <p class="card-margin technology">Install</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            quiz
+            </span>
+            <p class="card-margin technology">Consult</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            point_of_sale
+            </span>
+            <p class="card-margin technology">Sales</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            cast_for_education
+            </span>
+            <p class="card-margin technology">Training</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+      </div>
+      <a href="" class="more-link">See The Whole Services We Offer</a>
+
+
+
+      <button class="btn">Sign up for business</button>
+
+    </div>
+
+    <div class="client-section">
+      <h4>What our clients say...</h4>
+      <div class="program-line"></div>
+
+      <!-- dynamic content -->
+
+
+      <div class="main-content"></div>
+    </div>
+
+    <section>
+      <div class="partners">
+        <h5 class="partner-header">Training Partners</h5>
+        <div class="program-line"></div>
+        <div class="partner-div">
+           <div class="partner-image">
+            <img src="assets/partner/amazonfinal.png" alt="" width="110px">
+            <img src="assets/partner/cisco.png" alt="" width="110px" height="60px">
+            <img src="assets/partner/facebook.png" alt="" width="110px" height="50px">
+            <img src="assets/partner/google.png" alt="" width="110px" height="60p">
+           </div>
+        </div>
+      </div>
+    </section>
+
+    <div class="show">
+      <p>See More</p>
+      <div class="arrow">
+        <span class="material-symbols-outlined">
+        keyboard_arrow_down
```

Comment:
- [ ] Good job trying to removing the show LESS button :clap: :clap:, however as requested by the previous reviewer, while you have been doing it you forgot to put it at the bottom ( please hide the partners' section and footer which must only be visible to the desktop version). 
- Your design:
![Ymore](https://user-images.githubusercontent.com/97834160/210791282-bbbefaea-d506-4612-8d2a-eab34c8a1254.png)

- Behance design:
![Bmore](https://user-images.githubusercontent.com/97834160/210791085-8d8003b9-47c6-4191-b7d2-066a58f10d78.png)